packages/react-menu/config/normalize-import.js

```diff
@@ -0,0 +1,96 @@
+// @ts-check
+
+/// <reference lib="es2019.array" />
+
+const fs = require('fs');
+const path = require('path');
+const { default: chalk } = require('chalk');
+const yargsParser = require('yargs-parser');
+
+/**
+ * @type {Console['log']}
+ */
+// eslint-disable-next-line no-console
+const log = console.log.bind(console);
```

Comment:
What's the motivation for doing this, rather than using console.log directly and disabling the lint rule for the file?

Reply:
- a good rule of thumb is to not turn off rules whole whole file
- `log` is shorter
- ideally we should use robust and common logging solution - like [debug](https://www.npmjs.com/package/debug)

Reply:
I see nothing wrong with turning off a rule which is clearly irrelevant for a given file (or ideally disabling this rule for all config files). Making an alias for a common method decreases clarity and is a micro-optimization, and IMO not really worth it especially if your main goal is just to get around a lint rule.

Reply:
The intent of the no-console rule is to minimize console logs in component/other browser code files. It shouldn't be enabled for config files to begin with. So the best immediate workaround (either in this PR or a follow-up) would be to update the shared eslint config to ensure that it uses appropriate rules on config files.